tap_sailthru/streams.py

```diff
@@ -467,38 +405,85 @@ def prepare_request_payload(
         Returns:
             A dictionary containing the request payload.
         """
-        return {'primary': 1}
+        return {"primary": 1}
 
     def get_child_context(self, record: dict, context: Optional[dict]) -> dict:
-        return {
-            "list_id": record["list_id"],
-            "list_name": record["name"]
-        }
+        return {"list_id": record["list_id"], "list_name": record["name"]}
 
     def parse_response(
-        self,
-        response: SailthruResponse,
-        context: Optional[dict]
+        self, response: SailthruResponse, context: Optional[dict]
     ) -> Iterable[dict]:
         """Parse the response and return an iterator of result rows."""
-        for row in response['lists']:
-            row['account_name'] = self.config.get('account_name')
+        for row in response["lists"]:
+            row["account_name"] = self.config.get("account_name")
             yield row
 
 
+class ListMembersParentStream(sailthruStream):
```

Comment:
Should the parent stream here be renamed to `SailthruStream`? `sailthruStream` is very JS